neurogym/wrappers/__init__.py

```diff
@@ -24,4 +21,4 @@
 
 
 def all_wrappers():
-    return sorted(list(ALL_WRAPPERS.keys()))
+    return sorted(ALL_WRAPPERS.keys())
```

Comment:
Actually, `sorted` returns a list anyway, so this would still return a list. However, your commented made me realize that this is the same as `sorted(D)` as well:

```py
D = {'C': 123, 'A': 65}
sorted(D.keys()) == sorted(list(D.keys())) == sorted(D)
```
> True